container/root/scripts/install_goss.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# Use goss for serverspec-like testing
+# @see https://goss.rocks
+#
+# Downloads, verifies, and installs
+# Requires curl and sha256sum to be present
+
+GOSS_VERSION=v0.3.6
```

Comment:
@bryanlatten Is there any particular reason to not keep these in the Dockerfile and use an env var here instead?

Reply:
@sravankasu I considered that, and it's certainly an open question. The most pressing issue is that the value (`v.0.3.6`) would then be duplicated across all Dockerfiles

Reply:
I see your point, I agree it's easier to manage this way. If downstream users aren't affected I'm ok with this.

Reply:
@sravankasu thanks for the question. In fact, since we started using multi-phase builds, that environment variable hasn't been present. No downstream impact.